de-lib/src/main/java/org/iplantc/de/diskResource/client/views/search/cells/DiskResourceQueryForm.java

```diff
@@ -1,344 +1,65 @@
-import org.iplantc.de.client.models.tags.Tag;
-import org.iplantc.de.client.util.SearchModelUtils;
-import org.iplantc.de.commons.client.info.ErrorAnnouncementConfig;
-import org.iplantc.de.commons.client.info.IplantAnnouncer;
-import org.iplantc.de.commons.client.widgets.IPlantAnchor;
-import org.iplantc.de.diskResource.client.events.search.SaveDiskResourceQueryClickedEvent;
-import org.iplantc.de.diskResource.client.events.search.SubmitDiskResourceQueryEvent;
-import org.iplantc.de.diskResource.client.events.search.SubmitDiskResourceQueryEvent.HasSubmitDiskResourceQueryEventHandlers;
-import org.iplantc.de.diskResource.client.events.search.SubmitDiskResourceQueryEvent.SubmitDiskResourceQueryEventHandler;
-import org.iplantc.de.tags.client.TagsView;
-import org.iplantc.de.tags.client.events.TagAddedEvent;
-import org.iplantc.de.tags.client.events.selection.RemoveTagSelected;
+import org.iplantc.de.commons.client.util.CyVerseReactComponents;
+import org.iplantc.de.diskResource.client.SearchView;
+import org.iplantc.de.diskResource.client.views.search.ReactSearchForm;
 
-import com.google.common.base.Strings;
-import com.google.common.collect.Lists;
-import com.google.gwt.core.client.GWT;
+import com.google.gwt.dom.client.DivElement;
 import com.google.gwt.dom.client.Element;
-import com.google.gwt.editor.client.Editor;
-import com.google.gwt.editor.client.SimpleBeanEditorDriver;
-import com.google.gwt.editor.client.adapters.SimpleEditor;
-import com.google.gwt.event.dom.client.ClickEvent;
-import com.google.gwt.event.dom.client.ClickHandler;
 import com.google.gwt.event.dom.client.KeyCodes;
-import com.google.gwt.event.shared.HandlerRegistration;
-import com.google.gwt.safehtml.shared.SafeHtml;
-import com.google.gwt.uibinder.client.UiHandler;
 import com.google.gwt.user.client.Event;
-import com.google.gwt.user.client.Event.NativePreviewEvent;
-import com.google.gwt.user.client.ui.HorizontalPanel;
-import com.google.gwt.user.client.ui.Label;
 import com.google.gwt.user.client.ui.RootPanel;
-import com.google.gwt.user.client.ui.VerticalPanel;
 import com.google.inject.Inject;
 
-import com.sencha.gxt.cell.core.client.form.ComboBoxCell.TriggerAction;
-import com.sencha.gxt.core.client.Style.Anchor;
-import com.sencha.gxt.core.client.Style.AnchorAlignment;
-import com.sencha.gxt.core.client.XTemplates;
+import com.sencha.gxt.core.client.Style;
 import com.sencha.gxt.core.client.dom.XElement;
 import com.sencha.gxt.core.client.util.BaseEventPreview;
-import com.sencha.gxt.core.client.util.DateWrapper;
-import com.sencha.gxt.data.shared.LabelProvider;
 import com.sencha.gxt.widget.core.client.Composite;
-import com.sencha.gxt.widget.core.client.button.TextButton;
-import com.sencha.gxt.widget.core.client.container.AbstractHtmlLayoutContainer.HtmlData;
-import com.sencha.gxt.widget.core.client.container.HtmlLayoutContainer;
+import com.sencha.gxt.widget.core.client.container.VerticalLayoutContainer;
```

Comment:
I still have some Sencha imports in here mostly to have the `DiskResourceSearchCell` be able to interact with the form.  Once we replace `DiskResourceSearchField`, these can all go away.